rplugin/python3/denite/ui/default.py

```diff
@@ -141,7 +141,7 @@ def init_buffer(self):
             # Create new buffer
             self._vim.call(
                 'denite#util#execute_path',
-                'silent ' + self._context['direction'] + ' new', '[denite]')
+                'silent keepalt ' + self._context['direction'] + ' new', '[denite]')
```

Comment:
A lint say the line is too long. The line should be

```python
    'silent keepalt %s new' % self._context['direction'],
    '[denite]'
)
```

Or something similar I guess.

By the way, I agree with your change while denite is not a file-like buffer :+1: